params/dhcpd-server.conf

```diff
@@ -3,6 +3,6 @@
 # Add subnet declaration for the server
 # Define range of addresses that the server can assign to other devices
 subnet 192.168.2.0 netmask 255.255.255.0 {
-	range 192.168.2.3 192.168.2.254;
+	range 192.168.2.1;
```

Comment:
What happens if someone connects a different computer to it?
I had issues (in another setup) where there were the dhcp server had ran out of ips to lease and just stopped leasing new ips.

Reply:
We support exactly one client on exactly one ip address right now.

If we lease other ips, they will not get a video stream.

The dhcp with only one client still reduces user side configuration, we may support multiple clients in the future.

Reply:
Sorry I wasnt clear.
I mean what happens if the user switches to another computer? Is the ip reassigned to the new computer?

Reply:
If someone puts a second computer in the network with a static ip "192.168.2.3", can this ip be  detected by companion ?

Reply:
Again, the problem I had was that the number of "leasable" ips expired, so when a usertried to use another computer, the dhcp server didn't have any ips available and the user could not connect. this should be just a matter of dhcpd config(maybe lease time?).

Reply:
> I mean what happens if the user switches to another computer? Is the ip reassigned to the new computer?

Not until companion is rebooted. I tried to make a short lease time to get around this, but it did not work over here. I don't think this is a use case for concern. 

> If someone puts a second computer in the network with a static ip "192.168.2.3", can this ip be detected by companion ?

Yes

Reply:
if the lease expires when companion reboots it should be ok.